docs/development.md

```diff
@@ -21,67 +21,77 @@ The following list illustrates key features of the plugin system in ImJoy:
 ### ImJoy architecture
 Imjoy consists of **two main components**
 
- 1.  The **ImJoy Web App**. The app can run alone, and plugins can be developed in
-     JavaScript or in Python by using [pyodide](https://github.com/iodide-project/pyodide).
-
- 2.  Complex computational tasks can be implemented in plugins, which run in the
-     **Plugin Engine**. The latest release of the plugin engine is available
-     together with installation  instructions on [GitHub](https://github.com/oeway/ImJoy-App/releases).
-     The plugin engine will try to upgrade itself from GitHub when it starts.
-     Packages are managed by `conda` and `pip` which provides access to the
-     entire Python ecosystem.
+ 1.  The **ImJoy Web App**. The core part of ImJoy which runs in the browser across different OS and devices. It provide a flexible plugin system with workflow and window managments. Plugins can be developed in JavaScript or in Python are saved directly in the browser database. Web plugins are running inside a `iframe` or a `webworker`, therefore developers can in principle use any frontend framework or javascript libraries independently for each plugin. 
```

Comment:
It provides a flexible plugin system with workflow and window management. Plugins can be developed in Javascript or in Python, and are saved directly in the browser database. Web plugins run inside an `iframe` or a `webworker`, therefore developers can in principle use any frontend framework or Javascript libraries independently for each plugin.

Reply:
OK, should be fixed now, thanks.